ldap/servers/slapd/add.c

```diff
@@ -818,6 +818,7 @@ op_shared_add(Slapi_PBlock *pb)
     slapi_ch_free((void **)&pwdtype);
     slapi_ch_free_string(&proxydn);
     slapi_ch_free_string(&proxystr);
+    delete_passwdPolicy(&pwpolicy);
```

Comment:
I don't think this is needed (false positive).   new_passwdPolicy() stores the new passwordPolicy object in the pblock, and when the pblock is destroyed so is this passwordPolicy struct.

Reply:
I did not realized that new_passwdPolicy was registering the policy in the pblock,
 so I will have to double check all delete_passwdPolicy change I have done and use /* coverity[leaked_storage] */ instead.

Reply:
Little history on this: the original work was done in:  https://github.com/389ds/389-ds-base/issues/147  Basically in every single operation we would allocate the password policy struct like 3-4 times :-)  So we ended up storing it in the pblock to save from doing all these duplicate allocations.